core/src/worker/activities.rs

```diff
@@ -42,12 +51,16 @@ use temporal_sdk_core_protos::{
         workflowservice::v1::PollActivityTaskQueueResponse,
     },
 };
-use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender};
-use tokio::sync::{Mutex, Notify};
+use tokio::sync::{
+    mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender},
+    Mutex, Notify,
+};
 use tokio_stream::wrappers::UnboundedReceiverStream;
 use tokio_util::sync::CancellationToken;
 use tracing::Span;
 
+type OutstandingActMap = Arc<DashMap<TaskToken, RemoteInFlightActInfo>>;
```

Comment:
There's a bunch of other places we use this abbreviation (in this same file... in this same line even). I think it's about as common as WFT  - there are some things for which I'd really like to be able to use shorthand and I think this is probably one that qualifies.

Reply:
I know, you probably noticed that I started changing that in my PRs.
I'm requesting this because this is the only codebase we have in the team where we use these abbreviations and I personally would like to see this change mostly for the reason I mentioned above.
If you feel that strongly about it, I'm fine keeping as is, ultimately it's your decision.

Reply:
The way I look at it is it'll take a reader a few seconds to figure this out (we're in a file named `activities` after all), but the cumulative line length additions leading to more line breaks etc are probably more harmful than the tiny warmup time it takes to learn the two abbreviations.

For anything in the public API, I definitely agree